react/example-customer-account--loyalty-app--react/extensions/customer-account-loyalty-extension-profile/shopify.extension.toml

```diff
@@ -0,0 +1,33 @@
+# Learn more about configuring your Customer account UI extension:
+# https://shopify.dev/api/customer-account-ui-extensions/unstable/configuration
+
+# The version of APIs your extension will receive. Learn more:
+# https://shopify.dev/docs/api/usage/versioning
+api_version = "unstable"
+
+[[extensions]]
+type = "ui_extension"
+name = "customer-account-loyalty-extension-profile"
+handle = "ca-loyalty-extension-profile"
+
+# Controls where in Shopify your extension will be injected,
+# and the file that contains your extension’s source code. Learn more:
+# https://shopify.dev/docs/api/customer-account-ui-extensions/unstable/extension-targets-overview
+
+# [START config.setup-targets]
```

Comment:
Why do we have START end END comments for the targets, but not the other blocks?

I'd just remove them everywhere honestly

Reply:
I believe this may used by Shopify.dev. Kenza actually added this in what she shipped Im just copying it over :D